### ISSUE TITLE
"Unpublished" styles in pages listings are very hard / impossible to see

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#5370]
### Issue Summary

In listing pages, Wagtail has styles for pages that are not published:

<img width="1155" alt="unpublished-styles" src="https://user-images.githubusercontent.com/877585/59192318-f5a44d00-8b79-11e9-8ebc-cf965acbeb84.png">

These are very hard to see – I’m not sure what value they bring, whether they should be removed altogether or switch to a visual indicator that is less subtle, and doesn’t rely on color alone.

### Steps to Reproduce

1. Go to the CMS admin
2. Create a page as draft, without publishing
3. Go to a listing view showing that page

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: no

### Technical details

* Wagtail version: 2.5a0, 2.6a0
* Browser version: latest Chrome

**Comment by cfarm** [comment]
Here are some options for restyling that:

# Italicize the draft page title
![Screen Shot 2019-07-26 at 4 26 36 PM](https://user-images.githubusercontent.com/702526/61979800-d8f59480-afc2-11e9-8e66-c3486909be05.png)

# Add a background that matches the "Draft" status badge to the whole row to make the status more obvious
![Screen Shot 2019-07-26 at 4 30 35 PM](https://user-images.githubusercontent.com/702526/61979798-d85cfe00-afc2-11e9-99a1-fc6a00e823f2.png)

# Add background and italicize the title

![Screen Shot 2019-07-26 at 4 29 56 PM](https://user-images.githubusercontent.com/702526/61979799-d85cfe00-afc2-11e9-8660-a48e9c8316ed.png)

**Comment by thibaudcolas** [comment]
👍 I vote for "Italicize". It seems enough for the purpose of this issue!

**Comment by saevarom** [comment]
As mentioned in #5776, after discussion at Bristol sprint, we agreed that having the whole row greyed out using the diagonal line background is more visually pleasing. The proposed solution in #5776 looks like this:

<img width="1013" alt="Screenshot 2020-01-23 at 12 30 59" src="https://user-images.githubusercontent.com/143557/72985955-08088980-3ddf-11ea-8fa4-9d334d2703ed.png">

**Comment by thibaudcolas** [comment]
We’ve discussed this with @benenright and agreed to simply stop highlighting unpublished pages in a special way. The "draft" status tag makes it very clear which pages aren’t published, and we don’t think this needs further calling to attention.

With this in mind, I believe the only change needed is removal of:

https://github.com/wagtail/wagtail/blob/b7ab0496f12f6133e363ef5f0cad2e7b01f5af54/client/scss/components/_listing.scss#L358-L360

I’ll tag this as a `good first issue` if anyone wants to take this on – the biggest difficulty here will just be setting up of Wagtail for local development.

**Comment by zerolab** [comment]
Fixed in #12043
